src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -1680,57 +1680,57 @@ New_%0_database=Nouvelle_base_%0
 New_%0_database_created.=Nouvelle_base_%0_créée.
 
 No_entry_found_for_IBSN_%0_at_www.ebook.de=Aucune_entrée_trouvée_pour_l'ISBN_%0_depuis_www.ebook.de
-Save_actions=
-Enable_save_actions=
-Always_reformat_.bib_file_on_save_and_export=
-
-Default_bibliography_mode=
-
-
-Show_only_preferences_deviating_from_their_default_value=
-default=
-key=
-type=
-value=
-
-Show_preferences=
-
-Enter_field_name_(e.g.,_title,_author)=
-
-Other_fields=
-Show_remaining_fields=
-
-link_should_refer_to_a_correct_file_path=
-
-abbreviation_detected=
-wrong_entry_type_as_proceedings_has_page_numbers=
-Copy_special=
-
-Abbreviate_journal_names=
-Abbreviating...=
-Adding_fetched_entries=
-Are_you_sure_you_want_to_remove_the_already_existent_SQL_DBs?=
-Display_keywords_appearing_in_ALL_entries=
-Display_keywords_appearing_in_ANY_entry=
-Fetching_entries_from_Inspire=
-Multiple_entries=
-None_of_the_selected_entries_have_BibTeX_keys.=
-Resolve_duplicate_keys=
-Unabbreviate_journal_names=
-Unabbreviating...=
-Usage=
-
-Are_you_sure_you_want_to_reset_all_settings_to_default_values?=
-Reset_preferences=
-
-Ill-formed_entrytype_comment_in_bib_file=
-Clipboard=
-Could_not_paste_entry_as_text\:=
-Do_you_still_want_to_continue?=
-Please_enter_the_desired_name\:=
-SQL_Export=
-This_action_will_modify_the_following_field(s)_in_at_least_one_entry_each\:=
-This_could_cause_undesired_changes_to_your_entries.=
-You_have_entered_an_invalid_or_already_existent_DB_name.=
-
-Disable_highlight_groups_matching_entries=
+Save_actions=Enregistrer_les_actions
+Enable_save_actions=Autoriser_l'enregistrement_des_actions
+Always_reformat_.bib_file_on_save_and_export=Toujours_remettre_en_forme_les_fichiers_.bib_lors_de_l'enregistrement_et_de_l'exportation
+
+Default_bibliography_mode=Mode_bibliographique_par_défaut
+
+
+Show_only_preferences_deviating_from_their_default_value=Afficher_uniquement_les_préférences_différentes_de_leur_valeur_par_défaut
+default=par_défaut
+key=clef
+type=type
+value=valeur
+
+Show_preferences=Afficher_les_préférences
+
+Enter_field_name_(e.g.,_title,_author)=Entrez_le_nom_de_champ_(p._ex._title,_author)
+
+Other_fields=Autres_champs
+Show_remaining_fields=Afficher_les_champs_restants
+
+link_should_refer_to_a_correct_file_path=Le_lien_doit_correspondre_à_un_chemin_de_fichier_valide
+
+abbreviation_detected=Abréviation_détectée
+wrong_entry_type_as_proceedings_has_page_numbers=Mauvais_type_d'entrée_car_le_proceedings_a_des_numéros_de_page
+Copy_special=Copie_spéciale
+
+Abbreviate_journal_names=Abrégez_les_noms_de_journaux
+Abbreviating...=Abrègement_en_cours
+Adding_fetched_entries=Ajout_des_entrées_récupérées
+Are_you_sure_you_want_to_remove_the_already_existent_SQL_DBs?=Etes-vous_sûr_de_vouloir_supprimer_une_base_SQL_existante_?
```

Comment:
Okay I just found out, that you french guys place a whitespace in front of question marks ;-)
So forget about my comments...